### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by albertcui**
Enjoy!

### CODE REVIEW

File: src/lang/ru-RU.json [hunk 1]
```diff
@@ -110,8 +109,8 @@
   "barracks_value_512": "Средний дальнего боя сил Света",
   "barracks_value_1024": "Верхний ближнего боя сил Света",
   "barracks_value_2048": "Верхний дальнего боя сил Света",
-  "benchmarks_description": "{0} {1} равен или больше {2}% недавних показателей на этом герое",
-  "fantasy_description": "{0} на {1} балла",
+  "benchmarks_description": "https://www.opendota.com/players/121943326",
```
Comment (Luchanso) [inline src/lang/ru-RU.json]: It's real correct?

File: src/lang/ru-RU.json [hunk 2]
```diff
@@ -87,8 +87,7 @@
   "app_pvgna": "Найти руководство",
   "app_pvgna_alt": "Найти руководство на Pvgna",
   "app_rivalry": "Поставить на про-матч",
-  "app_rivalry_team": "Ставки на {0} киберспорт",
-  "app_moremmr": "Научиться играть как профессионал",
+  "app_rivalry_team": "Https://www. opendota. com/players/121943326",
```
Comment (Luchanso) [inline src/lang/ru-RU.json]: It's correct?

File: src/lang/ru-RU.json (hunk 1 above)
Comment (howardchung) [reply src/lang/ru-RU.json]: Nope, someone has submitted bad translations for these. If you could provide a correct translation please do so!

Comment (Luchanso) [reply src/lang/ru-RU.json]: @howardchung where I can do that?

**Pull request merged**